wrapper_test.go

```diff
@@ -282,6 +285,11 @@ func (w *wrapperTestSuite) TestLambdaHandlerE2ELocal() {
 			assert.Equal(w.T(), "bd862e3fe1be46a994272793", startFuncSpan.TransactionID)
 			assert.Equal(w.T(), string(inputPayload), startFuncSpan.Event)
 			assert.Equal(w.T(), version, startFuncSpan.SpanInfo.TracerVersion.Version)
+			if i > 0 { // check if second test
+				assert.Equal(w.T(), "warm", startFuncSpan.LambdaReadiness)
+			} else {
+				assert.Equal(w.T(), "cold", startFuncSpan.LambdaReadiness)
+			}
```

Comment:
I think we should have a variable `expectedReadiness` so it won't break easily. Same regarding the test of the endSpan.
With the current implementation, if we'll change the order of the tests then the test will break.